src/components/layout/navbar/index.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import Image from "../../base/image";
+import Styles from "./index.module.scss";
+
+function NavBar() {
+  return (
+    <nav
+      className={`container flex justify-between width-100 mx-auto ${Styles.navbar}`}
+    >
+      <div className="flex">
+        <Image src="logo.png" alt="logo" />
```

Comment:
Could we use const for alt value and import it?
```suggestion
        <Image src="logo.png" alt={LOGO} />
```